web/src/components/assets/Assets.tsx

```diff
@@ -1,59 +1,153 @@
+      if (errorData?.error_message) {
+        throw new Error(errorData.error_message);
+      }
+    }
+
+    // Refresh assets list after successful creation
+    const updatedAssetsResult = await getStudiesByStudyIdAssets({ path: { studyId } });
+    if (updatedAssetsResult.response.status === 200 && updatedAssetsResult.data) {
+      setStudyAssets(updatedAssetsResult.data);
+      if (setAssetManagementCompleted) {
+        setAssetManagementCompleted(true);
+      }
+      setShowAssetForm(false);
+    }
+  };
+
+  if (isLoading) return null;
 
   return (
-    <>
-      {/* <div className={styles["selected-study-header"]}>
-        <div className={styles["study-breadcrumb"]}>
-          <span>
-            Study: <strong>{selectedStudy?.title}</strong>
-          </span>
-
-          <Button variant="tertiary" size="small" onClick={() => setSelectedStudy(null)}>
-            Change Study
-          </Button>
+    <section className={styles["study-assets-container"]} data-cy="study-assets">
+      <h2 className="subtitle">Study Assets for study: {studyTitle}</h2>
+
+      {error && (
+        <div className={styles["error-message"]}>
+          <strong>Error:</strong> {error}
+        </div>
+      )}
+
+      <Callout definition>
+        <div className={styles["callout-section"]}>Use this form to create assets linked to your study.</div>
+
+        <div className={styles["callout-info-paragraph"]}>
+          Assets are any kind of data or information entity (e.g. consent forms, physical study materials etc.). They
+          are owned by a <strong>study</strong>{" "}
+          <InfoTooltip text="Studies are a top level entity that can contain own and projects" /> and can belong to{" "}
+          <strong>projects</strong> <InfoTooltip text="Projects are owned by a study and can contain assets" />.
         </div>
-      </div>
 
-      <AssetForm onSubmit={handleAssetSubmit} setSelectedStudy={setSelectedStudy} /> */}
-      Placeholder
-    </>
+        <div className={styles["callout-glossary-section"]}>
+          You can read more detailed information about assets in our{" "}
+          <Button href="/glossary" variant="tertiary" size="small" inline>
+            Glossary
+          </Button>{" "}
+          and the{" "}
+          <a
+            href="http://www.nationalarchives.gov.uk/documents/information-management/information-assets-factsheet.pdf"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            The National Archives guide on information assets.
+          </a>
+        </div>
+      </Callout>
+
+      {studyAssets.length === 0 ? (
+        <AssetCreationForm handleAssetSubmit={handleAssetSubmit} />
+      ) : (
+        <div>
+          <div className={styles["assets-summary"]}>
+            <span>Assets for this study:</span>
+            <span className={styles["assets-count-badge"]}>{studyAssets.length}</span>
+          </div>
```

Comment:
I'm not sure this is that useful(?)

Reply:
i vote keep unless we're told it's not helpful